addons/payment_worldline/models/payment_transaction.py

```diff
@@ -19,6 +19,17 @@
 class PaymentTransaction(models.Model):
     _inherit = 'payment.transaction'
 
+    def _compute_reference(self, provider_code, prefix=None, separator='-', **kwargs):
+        ref = super()._compute_reference(provider_code, prefix=prefix, separator=separator, **kwargs)
+        if provider_code != 'worldline':
+            return super()._compute_reference(provider_code, prefix=prefix, separator=separator, **kwargs)
```

Comment:
```suggestion
            return super()._compute_reference(
                provider_code, prefix=prefix, separator=separator, **kwargs
            )
```
Nitpicking: we try to stay < 100 chars per line =]